server/routes/visit.test.ts

```diff
@@ -55,6 +55,29 @@ jest.mock('./visitorUtils', () => {
   }
 })
 
+const availableSupportTypes: SupportType[] = [
```

Comment:
For now just copied this over from `bookAVisit.test.ts` - but a prime candidate for moving elsewhere to better handle test data (a future PR!)